File: app/src/main/java/com/wolcano/musicplayer/music/mvp/listener/OnServiceListener.java
package com.wolcano.musicplayer.music.mvp.listener;


import com.wolcano.musicplayer.music.mvp.models.Song;

public interface OnServiceListener {

    void onBufferingUpdate(int percent);

    void onChangeSong(Song song);

    void onPlayStart();

    void onPlayPause();

    void onProgressChange(int progress);


}

File: app/src/main/java/com/wolcano/musicplayer/music/mvp/listener/OnServiceListener.kt
package com.wolcano.musicplayer.music.mvp.listener

import com.wolcano.musicplayer.music.model.Song

interface OnServiceListener {
    fun onBufferingUpdate(percent: Int)
    fun onChangeSong(song: Song?)
    fun onPlayStart()
    fun onPlayPause()
    fun onProgressChange(progress: Int)
}